src/section/member_info.rs

```diff
@@ -94,6 +94,10 @@ impl MemberInfo {
         self.age_counter >= AgeCounter(2_u32.pow(MAX_INFANT_AGE + 1))
     }
 
+    pub fn is_new_adult(&self) -> bool {
+        self.age_counter == AgeCounter(2_u32.pow(MAX_INFANT_AGE + 1))
```

Comment:
why not using the following `age_counter_value` method?
so that no need to construct a AgeCounter